spec/package_spec.rb

```diff
@@ -293,6 +287,11 @@ def build_app_source_with_files(files)
     it 'ignores "description"' do
       expect(subject['description']).to be nil
     end
+
+    it 'preserves the rest' do
+      puts subject
+      expect(subject['everything_else']).to eq 'preserved'
```

Comment:
maybe, but I think its better to debug failed assertions than KeyError raised in test